meshtastic/tests/test_mesh_interface.py

```
@@ -7,25 +7,27 @@
 import io
 import logging
 import re
+import sys
 import threading
 import time
 import types
 from collections import OrderedDict
 from pathlib import Path
-from typing import TYPE_CHECKING, Any, cast
-from unittest.mock import MagicMock, create_autospec, patch
+from typing import TYPE_CHECKING, Any, Callable, NoReturn, cast
```

Ruff (0.15.5)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)